### ISSUE TITLE
Error thrown with Node.js 14

### ISSUE THREAD

**Issue by fivdi** [fivdi/onoff#170]
With Node.js 14 invoking `exportGpio` results in an error being thrown [here](https://github.com/fivdi/onoff/blob/v5.0.1/onoff.js#L18). This happens because `fs.writeFileSync` no longer accepts a number as data.

```js
pi@raspberrypi:~/junk/node_modules/onoff/examples $ node blink-led.js 
internal/fs/utils.js:656
    throw new ERR_INVALID_ARG_TYPE(
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received type number (17)
    at Object.writeFileSync (fs.js:1380:5)
    at exportGpio (/home/pi/junk/node_modules/onoff/onoff.js:18:8)
    at new Gpio (/home/pi/junk/node_modules/onoff/onoff.js:172:36)
    at Object.<anonymous> (/home/pi/junk/node_modules/onoff/examples/blink-led.js:4:13)
    at Module._compile (internal/modules/cjs/loader.js:1185:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1205:10)
    at Module.load (internal/modules/cjs/loader.js:1034:32)
    at Function.Module._load (internal/modules/cjs/loader.js:923:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'ERR_INVALID_ARG_TYPE'
}
```

**Comment by fivdi** [comment]
Fixed by bf93550cccde0c568bee586894d0728c3702b318

**Comment by fivdi** [comment]
The fix for this issue is available in onoff@6.0.0
